### PR TITLE
fix(EditTask):Fixed Sprint Data Changes

### PR DESCRIPTION
**Pull request by sanjaykrishna1203**
### Functionality:
Fix Sprint Data Changes on Edit Task

### Solution:
Removed the condintions with Status Labels.
Instead Used the Current Sprint Value for comparison

### Risk level:
- [x] high 
- [ ] medium
- [ ] low

### How to test:
Try Editing Tasks from one sprint to another
Try Editing the Story points

### CODE REVIEW

File: functions/model/sprints/tark/updateSprint.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
+const { getSprint, updateSprint, setSprint } = require("../lib");
+
+exports.updateSprintData = function (teamId, project, orgDomain, previousSprintName, oldStoryPointNumber, storyPointNumber, editedSprintNumber, orgId, editedSprintName, currentSprint) {
+    const sprintPromises = []
```
Comment (viv9k) [inline functions/model/sprints/tark/updateSprint.js:4]: This will throw eslint issues

**Review by viv9k — CHANGES_REQUESTED** [review]
Please do eslint fix

File: functions/model/sprints/tark/updateSprint.js (hunk 1 above)
Comment (sanjaykrishna1203) [reply functions/model/sprints/tark/updateSprint.js]: Resolved

**Review by viv9k — APPROVED** [review]
Approved

**Pull request merged**